### ISSUE TITLE
Please publish this on the logseq marketplace

### ISSUE THREAD

**Issue by menelic** [sawhney17/logseq-discourse-graphs#5]
such an important plugin belongs in the marketplace ;-)

**Comment by jaker-dotcom** [comment]
it is now

**Comment by menelic** [comment]
yes, but it has not been updated in a long time, through @sawhney17 did tease some great, almost done functionality - teh much needed ability to name and define connection types - on the logseq discord some time ago
